source/about/licensing-and-subscription.rst

```diff
@@ -185,7 +185,7 @@ Yes, once your payment is successfully processed your license is immediately ava
 How will I know when to renew my license?
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-You will be notified 60 days prior to your license expiry that your license is due for renewal, via a blue banner displayed at the top of your Mattermost window. This banner is only visible to System Admins.
+You will be notified 60 days before your license expiry that your license is due for renewal, via a blue banner displayed at the top of your Mattermost window. This banner is only visible to System Admins.
```

Comment:
```suggestion
You will be notified 60 days prior to your license expiry that your license is due for renewal, via a blue banner displayed at the top of your Mattermost window. This banner is only visible to System Admins.
```

Reply:
We say "prior to" as it refers specifically to the event that follows, whereas "before" is more general.